examples/gno.land/r/gh/issues_prs.gno

```diff
@@ -0,0 +1,18 @@
+package gh
+
+/*
+// IssueOrPR represents a GitHub issue or pull request
+type IssueOrPR struct {
```

Comment:
For this, I think we should have `type Issue` and `type PR`, which embeds Issue, Issue implements `type Referencer interface { Reference() string }`, returns `"#1134"`

(feel free to ignore, I think the comment makes it clear that you want to tackle this in another PR)